suites/pacific/rbd/tier-3_rbd_cache_scenarios.yaml

```diff
@@ -0,0 +1,99 @@
+# Tier3: Extended RBD cache scenario tests.
+#
+# This test suite runs addition test scripts to evaluate the existing functionality of
+# Ceph RBD component involving various cache scenarios.
```

Comment:
Since it is currently only for dm-cache scenario, maybe you can mention the same here. This sentence might feel misleading.

Reply:
Done